plugins/GSdx/GSSetting.cpp

```diff
@@ -124,10 +124,10 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_LOGZ:
 			return "Treat depth as logarithmic instead of linear. Recommended setting is on unless it causes graphical glitches.";
 #endif
-#ifdef __linux__
 		case IDC_MIPMAP:
 			return "Enables mipmapping, which some games require to render correctly. Turn off only for debug purposes.";
-#endif
+		case IDC_PRELOAD_GS:
+			return "The Following Hack uploads GS data when rendering a new frame to reprdouce some effects accurately, Fixes Black Screen issues in games like Primal. ";
```

Comment:
Added Armored core: Last Raven to the tool tip instead of Primal, since Primal was only fixed on replay of dump while using ``Preload Data Frame`` not while playing the game itself.